src/Extension/Traits/FluentObjectTrait.php

```diff
@@ -108,6 +113,37 @@ protected function updateFluentCMSFields(FieldList $fields)
             'Locale' => 'Locale'
         ];
 
+        // Augment Localisation tab with clickable locale links to allow easy navigation between model localisations
+        if ($owner->hasExtension(CMSEditLinkExtension::class)) {
+            $controller = Controller::has_curr() ? Controller::curr() : null;
+            $request = $controller?->getRequest();
+
+            // Pass getVars separately so we can process them later
+            $params = $request?->getVars() ?? [];
+
+            // This is to get URL only, getVars are not part of the URL
+            $url = $owner->CMSEditLink();
+            $url = Director::makeRelative($url);
+
+            $summaryColumns['Title'] = [
+                'title' => 'Title',
+                'callback' => function (Locale $object) use ($url, $params): ?DBField {
+                    if (!$object->RecordLocale()) {
+                        return null;
+                    }
+
+                    $recordLocale = $object->RecordLocale();
+                    $locale = $recordLocale->getLocale();
+                    $params['l'] = $locale;
+                    $localeLink = Controller::join_links($url, '?' . http_build_query($params));
+                    $localeTitle = Convert::raw2xml($recordLocale->getTitle());
+                    $render = sprintf('<a href="%s" target="_top">%s</a>', $localeLink, $localeTitle);
+
+                    return DBField::create_field('HTMLVarchar', $render);
+                }
+            ];
+        }
+
         // Let extensions override columns
         $owner->extend('updateLocalisationTabColumns', $summaryColumns);
```

Comment:
Having the link added before this hook does change how the value is when it's passed into the extension hook - however, because it was manipulated in the extension hook in the past, the value would be different depending on the order of extension hooks being called.

This is now more consistent, so IMO it's better than it was before in that regards.